gulpfile.mjs

```diff
@@ -1959,6 +1959,14 @@ gulp.task("lint", function (done) {
     styleLintOptions.push("--fix");
   }
 
+  const prettierOptions = [
+    "node_modules/prettier/bin/prettier.cjs",
```

Comment:
Is there perhaps a `.mjs`-file we could use instead?

Reply:
Unfortunately this is the only file in the bin directory.